modules/apps/29-fee/keeper/msg_server.go

```diff
@@ -57,7 +57,9 @@ func (k Keeper) PayPacketFee(goCtx context.Context, msg *types.MsgPayPacketFee)
 func (k Keeper) PayPacketFeeAsync(goCtx context.Context, msg *types.MsgPayPacketFeeAsync) (*types.MsgPayPacketFeeAsyncResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	if err := k.EscrowPacketFee(ctx, msg.IdentifiedPacketFee.PacketId, msg.IdentifiedPacketFee); err != nil {
+	// TODO: Update MsgPayPacketFeeAsync to include PacketFee in favour of IdentifiedPacketFee
```

Comment:
This todo will be addressed in a follow up. I think the `MsgPayPacketFeeAsync` should look something like this:

```
message MsgPayPacketFeeAsync {
    ibc.core.channel.v1.PacketId packet_id = 1;
    PacketFee packet_fee = 2;
}
```

Reply:
Opened #975